### PR TITLE
Feature/request form

### PR DESCRIPTION
**Pull request by adventurini**
Created a playground for request forms to connect the API at /form endpoint.

Added error handling to neighborhood route and an error handling wrapper.

### CODE REVIEW

File: functions/helpers/firebase.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+const firebase = require("firebase");
+// Required for side-effects
+require("firebase/firestore");
+firebase.initializeApp({
```
Comment (hjhart) [inline functions/helpers/firebase.js]: Use env variables instead of hard coded strings. (e.g. `process.env.ENVIRONMENT_VARIABLE`)

Comment (adventurini) [reply functions/helpers/firebase.js]: Nice catch

File: .firebase/hosting.b3V0.cache [hunk 2]
```diff
@@ -1,61 +1,47 @@
-404.html,1584738636093,103a7773023c778e55712043648742edbe0bd24d63ff726bb9383276c612307e
```
Comment (hjhart) [inline .firebase/hosting.b3V0.cache]: I feel like this folder should be .gitignored

Comment (adventurini) [reply .firebase/hosting.b3V0.cache]: Agreed

**Pull request merged**